cryptographic_estimators/RankSDEstimator/ranksd_estimator.py

```diff
@@ -0,0 +1,145 @@
+               truncate (int): Truncate rather than round the output (default: 0).
+
+           Examples:
+               >>> from cryptographic_estimators.RankSDEstimator.ranksd_estimator import RankSDEstimator
+               >>> RSDE = RankSDEstimator(q=2, m=31, n=33, k=15, r=10, w=2)
+               >>> RSDE.table(show_all_parameters=1)
+               +---------------------+--------------------------------------------+
+               |                     |                  estimate                  |
+               +---------------------+-------+--------+---------------------------+
+               | algorithm           |  time | memory |         parameters        |
+               +---------------------+-------+--------+---------------------------+
+               | BasisEnumeration    | 206.0 |   17.6 |             {}            |
+               | OJ1                 | 160.6 |   16.5 |             {}            |
+               | OJ2                 | 204.9 |   15.9 |             {}            |
+               | GRS                 | 162.2 |   18.2 |             {}            |
+               | ImprovedGRS         | 147.2 |   18.0 |             {}            |
+               | GuessingEnhancedGRS | 138.3 |   18.1 |          {'t': 1}         |
+               | HybridLinearization | 174.4 |   12.4 |         {'t': 15}         |
+               | MaxMinors           | 153.0 |   33.0 |     {'a': 12, 'p': 2}     |
+               | SupportMinors       | 155.1 |   40.1 | {'b': 1, 'a': 11, 'p': 0} |
+               +---------------------+-------+--------+---------------------------+
+
+           Tests:
+
+               >>> from cryptographic_estimators.RankSDEstimator.ranksd_estimator import RankSDEstimator
+               >>> from cryptographic_estimators.RankSDEstimator.RankSDAlgorithms import BasisEnumeration,GRS,GuessingEnhancedGRS,OJ1,OJ2,HybridLinearization
+               >>> RSDE = RankSDEstimator(q=2, m=31, n=33, k=15, r=10, w=2, excluded_algorithms=[BasisEnumeration,GRS,GuessingEnhancedGRS,OJ1,OJ2,HybridLinearization])
+               >>> RSDE.table(show_all_parameters=1)
+               +---------------+--------------------------------------------+
+               |               |                  estimate                  |
+               +---------------+-------+--------+---------------------------+
+               | algorithm     |  time | memory |         parameters        |
+               +---------------+-------+--------+---------------------------+
+               | ImprovedGRS   | 147.2 |   18.0 |             {}            |
+               | MaxMinors     | 153.0 |   33.0 |     {'a': 12, 'p': 2}     |
+               | SupportMinors | 155.1 |   40.1 | {'b': 1, 'a': 11, 'p': 0} |
+               +---------------+-------+--------+---------------------------+
+
+
+               >>> RSDE = RankSDEstimator(q=2, m=31, n=33, k=15, r=10, w=2, excluded_algorithms=[BasisEnumeration,GRS,OJ1,OJ2,HybridLinearization])
```

Comment:
Is this test already covered by the two above?

Reply:
It is already kind of covered. As you may note, the difference is that GuessingEnhancedGRS is not excluded. Should I remove it or leave it?